terraform/aks/variables.tf

```diff
@@ -100,7 +100,8 @@ variable paas_app_config_file { default = "workspace-variables/app_config.yml" }
 variable env_config {}
 
 variable "postgres_extensions" {
-  default = null
+  type      = list(string)
```

Comment:
As discussed, this is the same for all environments. The value can be hard coded in the terraform resource

Reply:
Has been hardcoded in databases.tf now